package.json

```
@@ -27,7 +27,7 @@
   },
   "homepage": "https://github.com/ckeditor/ckeditor5-dev#readme",
   "scripts": {
-    "test": "mocha packages/*/tests --recursive --timeout 5000",
+    "test": "mocha 'packages/*/tests/**/*.js' --recursive --timeout 5000",
```

```suggestion
    "test": "mocha 'packages/*/tests/**/*.js' --timeout 5000",
```

It seems that the `--recursive` option is not needed.